File: arrays-and-strings/01-is-unique/java/SolutionBruteForce.java
public class SolutionBruteForce {

    static boolean isUnique(String text) {
        boolean unique = true;

        for (int i = 0; i < text.length(); i++) {

            for (int j = i + 1; j < text.length(); j++) {

                if (text.charAt(i) == text.charAt(j)) {
                    unique = false;
                    break;
                }
            }

            if (!unique) {
                break;
            }
        }

        return unique;
    }

    public static void main(String[] args) {
        System.out.println(isUnique("LoremIpsum"));
    }
}

File: arrays-and-strings/01-is-unique/csharp/SolutionBruteForce.cs
using System;

public class Program
{
    static bool IsUnique(string text)
    {
        bool unique = true;

        for (int i = 0; i < text.Length; i++)
            {
                for (int j = i + 1; j < text.Length; j++)
                {
                    if (text[i] == text[j])
                    {
                        unique = false;
                        break;
                    }
                }

                if (!unique) break;
            }

        return unique;
    }

    static void Main(string[] args)
    {
        Console.Write(IsUnique("LoremIpsum"));
    }
}